_maps/lavaland.json

```diff
@@ -0,0 +1,32 @@
+{
+	"map_name":"Lavaland",
+	"map_path":"map_levels/140x140",
+	"map_file":[
+		 "lavaland.dmm",
```

Comment:
It may not change anything but perhaps reorder the way you load these. First load Dungeon and East, then load Lavaland.dmm. I am thinking that perhaps that by starting on a Z level that has no "up" or "down" it will load the maps as if they were flat.